src/index.js

```diff
@@ -1,34 +1,31 @@
 import 'bootstrap/dist/css/bootstrap.min.css';
-import './reset.css'
+import './reset.css';
 import './style.css';
-import * as Update from './Update'
-import Onclic from './onclic'
+import * as Update from './Update';
+import Onclic from './onclic';
 import * as helper from './style_helpers';
 
-const apikey ='e07023ef36e4fb48fc91efb895731682';
-helper.btn('btnFind', ()=>{
-  const cityname = document.getElementById('city').value;
-  const link = 'https://api.openweathermap.org/data/2.5/forecast?q='+cityname+'&appid='+apikey;
-  getData(link);
-}, 'empty');
-
-
-async function getData(nurl){
-  let response = await fetch(nurl ,{mode:"cors"});
+async function getData(nurl) {
+  const response = await fetch(nurl, { mode: 'cors' });
   if (response.ok) {
     const jsonfile = await response.json();
-    if(jsonfile.cod ==='200'){
+    if (jsonfile.cod === '200') {
       Onclic(jsonfile);
-      helper.unstyle('btnC','selected');
-      helper.style('btnC','toggle');
-      helper.style('btnF','selected');
+      helper.unstyle('btnC', 'selected');
+      helper.style('btnC', 'toggle');
+      helper.style('btnF', 'selected');
       Update.celsius(jsonfile);
     }
-
   } else {
-      alert('No found, try again');    
+    alert('No found, try again');
   }
 }
```

Comment:
- [ ] [REQUIRED] This async code need to have error handling, I understand right now is checking if the response is ``not-found`, but it should also handle the case where there's an error with the API (with a try/catch statement)
![image](https://user-images.githubusercontent.com/73254040/101059091-8384ac80-3553-11eb-98ef-dfdee80ca026.png)
[Source](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)